### ISSUE TITLE
Civilian improvements and suggestions

### ISSUE THREAD

**Issue by judgegritter** [LondonsBlueLine/tracker#460]
**Describe the bug**
The current civilian menu/way of life is not fit for purpose.

1. There is only one place to get your vehicle and any weapons which links in with the issue that you spawn in random places instead of at the vehicle place. This means either running for an hour or committing a crime (stealing a car) to get to where your vehicle actually is. Those that want to create background stories (actual RP) then this makes it impossible. 

2. It is pointless having multiple characters because EUP does not save per character. If you choose an outfit/features for one character when you then switch to a different character the EUP carries over so you have to start all over again. I'm aware there are obviously AI peds but it is still an unnecessary task and using AI peds doesn't make the character unique. 

3. It is not possible to add 'warning markers' or 'arrest history' to your record as a civilian. If I am starting with a new character that I want to have a history of assaulting Police or contagious issues that has many previous arrests - I can't. 

4. Adding a vehicle to a character is long winded as you have to already know the name of the vehicle - having to google the GTA 5 vehicles and picking one you think fits the character to obtain the name of it. 

**Additional context**
This is likely the route cause of all incidents involving the same names and vehicles. Makes RP very boring.

**Comment by judgegritter** [comment]
vMenu would be a suggestion I would use. It is very easily controllable with permissions and allows access to spawning and saving of EUP peds and vehicles.

**Comment by jxckUK** [comment]
> vMenu would be a suggestion I would use. It is very easily controllable with permissions and allows access to spawning and saving of EUP peds and vehicles.

vMenu will not be added to the server under any circumstances. We made this decision a long time ago for a variety of internal reasons.

However, you raise some valid and fair comments about the existing civilian system. I'll address them below:

1. I shall add a number of additional spawn points for vehicles including at the apartments/houses that you can spawn at as civilians. This will allow you to retrieve owned vehicles at a larger array of areas.

2. Civilian appearance should be tied to your civilian record, unfortunately this has yet to be developed and needs to be added. It's already on the roadmap and I'll accelerate its progress as best I can.

3. Switching to the "Police" dashboard and searching for your civilian will allow you to add tags and arrest histories. This is intended behaviour because civilians should not be able to amend their own internal police records. You may amend them in the police dashboard for new characters but adding the ability to amend it as a civilian wouldn't make sense.

4. Adding a vehicle through this method ensures that the ANPR system as well as other critical framework resources can correctly identify your vehicle. There are a list of vehicles you can select within the CAD by typing the start of any vehicle. In the future, when we have spare development capacity, we may add visual elements such as a dealership but at this time its low priority.

**Comment by jxckUK** [comment]
![image](https://user-images.githubusercontent.com/48030374/197338837-f2943846-55ad-41a9-93f8-21521c57130b.png)
 
 New civilian garages will be effective from next restart post merge.

**Comment by judgegritter** [comment]
Thank you for the response.

In relation to 3 - 
It would make perfect sense if you use the example I have provided thus being a new character wants a history with the Police. 
Unless I am mistaken it does not appear possible to use the Police dashboard to add that way unless you're already in the server on duty as Police. Which as said makes the process very lengthy for something so simple.

**Comment by jxckUK** [comment]
> Thank you for the response.
> 
> In relation to 3 - It would make perfect sense if you use the example I have provided thus being a new character wants a history with the Police. Unless I am mistaken it does not appear possible to use the Police dashboard to add that way unless you're already in the server on duty as Police as I can't go 'state 1'. Which as said makes the process very lengthy for something so simple.

Hmm... fair point - let me see what I can do.

**Comment by jxckUK** [comment]
I'm waiting on some external development questions to be answered before I can proceed fully with point 2 but I have done the internal development ready to implement.

Once we've confirmed it, we can push it to live, and each civilian will have unique appearance data.

**Comment by jxckUK** [comment]
I'm still awaiting some CAD changes but haven't forgot about the final items on this request, appearance data will be retrieved and set from the CAD and you will be able to add history and previous charges to new characters.

Will update when complete.

**Comment by jxckUK** [comment]
As the requestor is, unfortunately for us, no longer a member of the community I'll track this internally on our roadmap.
